<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="box_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="box_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>60</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
